restricted_site_access.php

```diff
@@ -730,7 +730,7 @@ public static function enqueue_settings_script() {
 	public static function enqueue_admin_script() {
 		$current_screen = get_current_screen();
 
-		if ( ! empty( $current_screen ) && 'plugins-network' !== $current_screen->id ) {
+		if ( ! empty( $current_screen ) && ! in_array( $current_screen->id, [ 'plugins-network', 'options-reading' ], true ) ) {
```

Comment:
Can you please review my PR at https://github.com/10up/restricted-site-access/pull/131 to see which approach we should use to fix the nonce issue?

Reply:
Looks good.